news/migrations/0014_auto__add_smsmessage.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
```

Comment:
As soon as I can get someone in IT to get basket on the new Python 2.7 generic cluster. We're still on 2.6 :disappointed: